app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -30,13 +30,18 @@ import scala.concurrent.Future
 @Singleton
 class BindingTariffClassificationConnector @Inject()(configuration: AppConfig, client: HttpClient) {
 
+  def update(c: Case)(implicit hc: HeaderCarrier): Future[Case] = {
+    // TODO Replace this
+    Future.successful(c)
+  }
+
   def getOneCase(reference: String)(implicit hc: HeaderCarrier): Future[Option[Case]] = {
     val url = s"${configuration.bindingTariffClassificationUrl}/cases/$reference"
     client.GET[Option[Case]](url)
   }
 
   def getCasesByQueue(queue: Queue)(implicit hc: HeaderCarrier): Future[Seq[Case]] = {
-    val queueId = if (queue.id == 1) "none" else queue.id
+    val queueId = if (queue.id == "1") "none" else queue.id
```

Comment:
could we set some constant self-explanatory for this "1" ?

Reply:
Are you telling me "1" is not a constant? ;)

Reply:
Done

Reply:
Indeed. is a space/time constant. XD